ansible_community_steering_committee.rst

```diff
@@ -0,0 +1,106 @@
+************************************
+Ansible Community Steering Committee
+************************************
+
+The Steering Committee’s role is to provide continuity, guidance, and
+suggestions to the Ansible, and to ensure delivery of the Ansible
```

Comment:
```suggestion
suggestions to the Ansible community, and to ensure delivery of the Ansible
```
wasn't sure if this was meant to be "the Ansible community" or just "Ansible"

Reply:
Good find! Thanks @briantist